rosette/h/BuiltinClass.h

```diff
@@ -47,7 +47,7 @@ class BuiltinClass
 {
     static int			nClasses;
     static BuiltinClass*	root;
-    static unsigned long*	counts;
+    static Word32*	counts;
```

Comment:
Let's do one space here, please. It looks like Greg tried to do column-alignment, but it's not even consistent anymore and it's a nightmare to maintain. Can we just go to single spaces?

Reply:
Might be worth running a modern-styled `indent` on whole corpus and checking that in before doing small updates.

Reply:
IOW, get rid of all the tabs and line things up appropriately.

Reply:
I think this is a good suggestion. Otherwise, I will be irritating AF in every code review picking nits on the indentation style.

Reply:
I agree with @kirkwood. I'd like to pretty-print the whole thing with no tabs and make it editor independent.

I don't think indent can handle C++ constructs, but most modern editors have built-in formatting that can.

Reply:
I think clang-format supports C++